### PR TITLE
Portfolio form2

### PR DESCRIPTION
**Pull request by The-Normal-One1**
Add the contact form into the Html file and also add its style into the style.css file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -238,53 +238,81 @@ <h2 class="heading-content">
...
+        id="email"
+        placeholder="Email address"
+        required
+      />
+      <textarea
+        name="message"
+        cols="30"
+        rows="10"
+        placeholder="Enter text here "
+        max-length="500"
```
Comment (awais-amjed) [inline index.html]: Kindly fix the typo here, `max-length` should be `maxlength` 🤗

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @The-Normal-One1 , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (The-Normal-One1) [reply index.html]: Wow, thanks for your comment.

**Review by sajid-munawar — APPROVED** [review]
### Hi @awais-amjed , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please make sure you add a descriptive summary. a detailed summary let the reader understand what kind of changes you have made.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**